backvendor/gosource.go

```diff
@@ -73,6 +73,96 @@ func FindExcludes(pth string, globs []string) ([]string, error) {
 	return excludes, nil
 }
 
+// FindGoSources looks for top-level projects at pth. If pth is itself
+// a top-level project, the returned slice contains a single *GoSource
+// for that project; otherwise immediate sub-directories are tested.
+// Files matching globs in excludeGlobs will not be considered when
+// matching against upstream repositories.
+func FindGoSources(pth string, excludeGlobs ...string) ([]*GoSource, error) {
```

Comment:
I've poked into this a bit more. path is a parameter name, shadowing the path module, unless the path module is needed and then the parameter changes to mainly just p because the context of the short function and its name makes it obvious what p is. There's one occurrence of pth from external contributor Jan Mercl, and a path_ from Googler Matthew Dempsky.  The path_ survived edits by Griesemer.

A wider search, for `\<[a-zA-Z][a-zA-Z0-9_]*_\>` finds lots of examples. new, var, error, constant, range, func, fmt, append, and else all exist with a trailing underscore so I think that seems the prevalent style in the Go repository.

Reply:
Thanks. Changed to 'path' and 'p' throughout this particular PR, no underscore seemed to be necessary. I'll keep that naming rule in mind for future changes.